packages/flutter/lib/src/foundation/_isolates_io.dart

```diff
@@ -10,6 +10,10 @@ import 'package:meta/meta.dart';
 import 'constants.dart';
 import 'isolates.dart' as isolates;
 
+export 'dart:async' show FutureOr;
```

Comment:
that seems to be a bug in the lint, FutureOr isn't actually in a public API as far as I can tell

Reply:
Actually `ComputeCallback` returns a `FutureOr`